src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -1,25 +1,28 @@
 package at.hannibal2.skyhanni.features.garden.composter
 
 import at.hannibal2.skyhanni.SkyHanniMod
+import at.hannibal2.skyhanni.data.SackAPI.fetchSackItem
 import at.hannibal2.skyhanni.data.model.ComposterUpgrade
 import at.hannibal2.skyhanni.events.*
 import at.hannibal2.skyhanni.features.bazaar.BazaarApi
 import at.hannibal2.skyhanni.features.garden.GardenAPI
 import at.hannibal2.skyhanni.features.garden.composter.ComposterAPI.getLevel
+import at.hannibal2.skyhanni.utils.*
+import at.hannibal2.skyhanni.utils.ItemUtils.getInternalName
 import at.hannibal2.skyhanni.utils.ItemUtils.name
-import at.hannibal2.skyhanni.utils.LorenzUtils
 import at.hannibal2.skyhanni.utils.LorenzUtils.addAsSingletonList
 import at.hannibal2.skyhanni.utils.LorenzUtils.addSelector
+import at.hannibal2.skyhanni.utils.LorenzUtils.chat
+import at.hannibal2.skyhanni.utils.LorenzUtils.clickableChat
```

Comment:
What's wrong with this?
But also, IntellIJ imported those. I can change it ig.
Should I also change it for the other LorenzUtils imports?

Also isn't it more efficient to only import `SackAPI.fetchSackItem` instead of importing all of `SackAPI` and then using `SackAPI` before the call?

Reply:
> What's wrong with this? But also, IntellIJ imported those. I can change it ig. Should I also change it for the other LorenzUtils imports?
> 
> Also isn't it more efficient to only import `SackAPI.fetchSackItem` instead of importing all of `SackAPI` and then using `SackAPI` before the call?

The performance difference would be very small and these functions are within an API/Util class for a reason. The way you have it makes it confusing and does not fit with the rest of the codebase